NavigationAdvancedSample/app/src/main/java/com/example/android/navigationadvancedsample/MainActivity.kt

```diff
@@ -52,7 +52,8 @@ class MainActivity : AppCompatActivity() {
      */
     private fun setupBottomNavigationBar() {
         val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottom_nav)
-
+            
+        // the graphiIds and bottom Navigation menu items should has the same id to works
```

Comment:
There is a typo in`graphiIds`

Reply:
also `should has` -> `should have` and `the same id` -> `the same ids`